src/components/player/AllButtons.tsx

```diff
@@ -0,0 +1,75 @@
+import {Song} from "../../data/Songs";
+
+type AllButtonsProps = {
+    songs:Song[],
+    currentSong:number,
+    setCurrentSong(id:number):void
+}
+
+const AllButtons = (props:AllButtonsProps) => {
```

Comment:
you can desctructure the props, so referencing them would be easier.
```suggestion
const AllButtons = ({songs, currentSong, setCurrentSong}) => {
```
and then you can reference them directly(instead of using the suffix 'prop.')

Reply:
Also AllButtons is a vague name :) how about `PlayerButton`?

Reply:
Also, As I think about the utility of the component, It makes sense to get other props(The idea is to keep only the minimal logic required for the component so it will be easier to reuse it). I suggest the props:
onPrevClick, onPlayCLikc, onPauserClick, ... 
This way the component which using this component, will be able to freely specify the logic which will be executing in each handle. and we won't need to pass the songs list and the current song (which is a bit out of the scope of this component).
I hope I am not confusing you, let me know if I did :)